arch/x86/kvm/vmx/vmx.c

```diff
@@ -517,6 +519,8 @@ static DEFINE_PER_CPU(struct list_head, loaded_vmcss_on_cpu);
 static DECLARE_BITMAP(vmx_vpid_bitmap, VMX_NR_VPIDS);
 #ifndef __PKVM_HYP__
 static DEFINE_SPINLOCK(vmx_vpid_lock);
+#else
+static pkvm_spinlock_t vmx_vpid_lock;
```

Comment:
We don't initialize vmx_vpid_lock?

It shouldn't cause problems for now, since __PKVM_SPINLOCK_UNLOCKED is zero anyway. But it's better to explicitly initialize it? (BTW like I noted in another comment, it would be nice to have DEFINE_PKVM_SPINLOCK for that.)